pytube/__main__.py

```diff
@@ -79,6 +80,7 @@ def __init__(
         self.player_config_args: Dict = {}  # inline js in the html containing
         self.player_response: Dict = {}
         # streams
+        self.publish_date: Optional[datetime] = None
```

Comment:
Python 3.6 is almost 4 years old. Does it make sense to support 3.5? What would be a case where this makes sense.

Reply:
Python 3.5 got its most recent (and final) update last month, and commands somewhere around 12% market share of python installs from what I can gather. I really do think that we should encourage users to upgrade where they can, but we can't ignore the fact that python 3.5 was in active development up until just last month, even if it is an older version. We also know that a part of the pytube userbase is using python 3.5 without the ability to upgrade, because we have a [PR](https://github.com/nficano/pytube/pull/638) saying as much

In the future, I agree that we should drop support, but providing this backwards compatibility right now doesn't really have a negative impact on us.

Reply:
Good explanation. I adapt my changes accordingly.